package.json

```diff
@@ -243,6 +243,8 @@
     ]
   },
   "dependencies": {
+    "@datapunt/asc-core": "^0.9.2",
```

Comment:
Is the addition of the `asc` packages necessary for this PR?

Reply:
na de merge van develop niet meer